luaui/Widgets/gui_info.lua

```diff
@@ -1061,14 +1062,14 @@ local function drawUnitInfo()
 				end
 			end
 			if dps then
-				dps = round(dps + (dps * (reloadTime / 100)), 0)
+				dps = round(dps / (1 - (reloadTime / 100)), 0)
```

Comment:
Seems we do then undo calculations here a bit. 

The `1 -` and ` * 100` on [line 1059](https://github.com/beyond-all-reason/Beyond-All-Reason/pull/644/files#diff-f9bcfe83362c1bdddf26052cc6b0605c1b1ee0f3e8e93ec57579e9857a418a88R1059) are both undone on both [line 1065](https://github.com/beyond-all-reason/Beyond-All-Reason/pull/644/files#diff-f9bcfe83362c1bdddf26052cc6b0605c1b1ee0f3e8e93ec57579e9857a418a88R1065) and [1072](https://github.com/beyond-all-reason/Beyond-All-Reason/pull/644/files#diff-f9bcfe83362c1bdddf26052cc6b0605c1b1ee0f3e8e93ec57579e9857a418a88R1072), and the `/ unitDefInfo[displayUnitDefID]` is undone on [line 1072](https://github.com/beyond-all-reason/Beyond-All-Reason/pull/644/files#diff-f9bcfe83362c1bdddf26052cc6b0605c1b1ee0f3e8e93ec57579e9857a418a88R1072). 

For clarity, while you're here and if you feel like it, could these redundant calculations be reduced?